app/javascript/src/components/Dashboard/Notes/Note/Footer.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+
+import { Button, Avatar, Typography } from 'neetoui'
+import { Clock } from 'neetoicons'
+
+const Footer = () => {
+
+  return (
+    <>
+      <div className='flex items-center justify-between'>
+          <Button
+            label="Getting started"
+            className="mt-3 mx-2"
+            style="secondary"
+          />
+          <span className='flex justify-between'>
+            <Clock className="mt-3 mx-1" />
+            <Typography style="body3" className="mt-4 mx-1">
+                Created 2 hours ago
```

Comment:
This should not be hardcoded. This should be fetched from the resource. Here the resource is `note`.